### DISCUSSION TITLE
Map edge

### DISCUSSION THREAD

**Discussion by mesochaser2425** [niels747/2D-Weather-Sandbox#41] [Ideas]
So i was thinking if you could bring back map edges, becouse when there is loop it seems like the maps never ends and for me personaly when there are lets say winds and the storms are moving it makes the atmosphere "messy"

**Comment by Meso22** [comment]
The atmosphere always looped even with the walls there

**Reply by mesochaser2425** [reply]
yeah right, but im saying that adding non looped atmosphere can change alot

**Reply by Meso22** [reply]
I already asked that same sort of question a while back but forgot what the answer was.
Here's a screenshot
![image](https://user-images.githubusercontent.com/105577122/209587169-26122ccb-96a0-413c-966e-01f97c697efe.png)

**Reply by niels747** [reply]
Well I get what you mean, but I don't know how to implement it. I could make an invisible wall at the edge which would block the airflow but let the light shine trough. The problem is that this creates unrealistic updrafts at the edges of the map, because any wind collides with the wall at the edge. I might create a branch and try to implement it.

**Reply by mesochaser2425** [reply]
i mean not a invisible wall but just a non reapeat map but something adding new air to the simulation, like you could say how hot is the air and how much moisture it has
than it just spawns it onto a map

**Reply by niels747** [reply]
Spawning new air is tricky...
I have tried it before but it was very unstable
